mk/checkconf.mk

```diff
@@ -31,6 +31,22 @@ define check-conf-h
 	fi
 endef
 
+define build-conf-in
+	$(q)set -e;						\
+	cnf="$(strip $(foreach var,				\
+		$(call cfg-vars-by-prefix,CFG_),		\
+		$(call cfg-make-variable,$(var))))";		\
+	mkdir -p $(dir $@);					\
+	echo "# auto-generated TEE configuration file" >$@.tmp; \
+	echo "# TEE version ${CFG_TEE_VERSION}" >>$@.tmp;	\
```

Comment:
Minor nitpick: when CFG_TEE_VERSION is not defined (the default), you get an empty string. Better use:
`echo "# TEE version: ${CFG_TEE_VERSION:-(undefined)}"`